code/modules/makeup/nailpolish.dm

```diff
@@ -0,0 +1,158 @@
+/obj/item/organ/external/var/datum/nail_polish/nail_polish
+
+/obj/item/weapon/nailpolish
+	name = "nail polish"
+	desc = "Nail polish, to paint your nails with. Or someone else's!"
+	icon = 'icons/obj/nailpolish_vr.dmi'
+	icon_state = "nailpolish"
+	var/colour = "#FF0000"
+	var/icon/top_underlay
+	var/icon/top_underlay_open
+	var/icon/color_underlay
+	var/icon/color_underlay_open
+	var/open = FALSE
+	drop_sound = 'sound/items/drop/glass.ogg'
+	pickup_sound = 'sound/items/pickup/glass.ogg'
+
+/obj/item/weapon/nailpolish/Initialize()
+	. = ..()
+	regenerate_icons()
+
+/obj/item/weapon/nailpolish/proc/regenerate_icons()
+	top_underlay = icon(icon, "top")
+	top_underlay_open = icon(icon, "top-open")
+	color_underlay = icon(icon, "color")
+	color_underlay.Blend(colour, ICON_MULTIPLY)
+	color_underlay_open = icon(icon, "color-open")
+	color_underlay_open.Blend(colour, ICON_MULTIPLY)
+	update_icon()
+
+/obj/item/weapon/nailpolish/proc/set_colour(var/_colour)
+	colour = _colour
+	regenerate_icons()
+
+/obj/item/weapon/nailpolish/attack_self(var/mob/user)
+	open = !open
+	to_chat(user, SPAN_NOTICE("You [open ? "open" : "close"] \the [src]."))
+	update_icon()
+
+/obj/item/weapon/nailpolish/update_icon()
+	. = ..()
+	underlays.Cut()
+	icon_state = "[initial(icon_state)][open ? "-open" : null]"
+	if (open)
+		underlays = list(color_underlay_open, top_underlay_open)
+	else
+		underlays = list(color_underlay, top_underlay)
+
+/obj/item/weapon/nailpolish/proc/get_polish(var/mob/living/carbon/human/target, var/bp)
+	var/static/forbidden_parts
+	if(!forbidden_parts)
+		forbidden_parts = BP_ALL - list(BP_L_HAND, BP_R_HAND, BP_L_FOOT, BP_R_FOOT)
```

Comment:
This is not necessary in static vars local to procs. The declaration only runs once:

This:
```
/obj/proc/do_something()
    var/static/list/testlist = list("one","two")
    world.log << testlist
```
Decompiles to:
```
// path: /obj/proc/do_something
// display/inheritance name: do something
// signature (WIP): /obj/proc/do_something()
	GETVAR subvar world field "log" // L3
	GETVAR global 67 /* testlist */
	OUTPUT
	END
```
You can see the static is actually secretly a global var. It's just a sort of 'namespaced' way of having global vars. It's identical in terms of how byond uses it, anyway.